lib/node_modules/@stdlib/stats/base/dnanmeanors/benchmark/benchmark.js

```diff
@@ -21,16 +21,30 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float64Array = require( '@stdlib/array/float64' );
 var pkg = require( './../package.json' ).name;
 var dnanmeanors = require( './../lib/dnanmeanors.js' );
 
 
 // FUNCTIONS //
 
+/**
+* Returns a random value or `NaN`.
+*
+* @private
+* @returns {number} random number or `NaN`
+*/
+function rand() {
+	if ( bernoulli( 0.2 ) ) {
```

Comment:
```suggestion
	if ( bernoulli( 0.8 < 1 ) ) {
```
just a minor change here, we'll try to keep things explicit to show what is being returned etc...